ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml

```diff
@@ -3908,7 +4332,16 @@ If the client requests a text body whose native format is HTML, and the size of
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-ASAIRS_R1368</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.2.24.1</ns1:Doc_Sect>
-		<ns1:Description>[In FileReference (Attachment)] In protocol version 16.0, the server will return the FileReference element along with the ClientId element (section 2.2.2.14) as child elements of the Attachment element in response to a Sync command request ([MS-ASCMD] section 2.2.2.20) that adds an attachment [either to a calendar item or] to a draft email item. </ns1:Description>
+		<ns1:Description>In protocol version 16.0, the server will return the FileReference element along with the ClientId element (section 2.2.2.14) as child elements of the Attachment element in response to a Sync command request ([MS-ASCMD] section 2.2.1.21) that adds an attachment either to a calendar item or to a draft email item. </ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
```

Comment:
adapter

Reply:
The similar RS below marked as Unverified in last RS update.
[In   FileReference (Attachment)] In protocol version 16.0, the server will return   the FileReference element along with the ClientId element (section 2.2.2.14)   as child elements of the Attachment element in response to a Sync command   request ([MS-ASCMD] section 2.2.2.20) that adds an attachment [either] to a   calendar item [or to a draft email item]. |   | Protocol | Both | Normative | Unverified
-- | -- | -- | -- | -- | --